lib/datalog/roletester/build.rs

```diff
@@ -0,0 +1,6 @@
+use std::io::Result;
+
+fn main() -> Result<()> {
+    prost_build::compile_protos(&["proto/access.proto"], &["proto/"])?;
```

Comment:
Hm, why not compile this proto file in Makefile's `make grpc` target like all other proto definitions?

Reply:
Just adding my thoughts here from a Rust perspective. This way is generally strongly preferred in Rust and officially it's the only "supported" way of doing things. One reason for this is that the generated code may be specialised to the host machine among other things.

Reply:
Gotcha. It's fine but can we at least not duplicate proto definitions and keep only one of access.proto or types.proto?